src/couch/src/couch_db_engine.erl

```diff
@@ -43,10 +43,10 @@
 -type epochs() :: [{Node::atom(), UpdateSeq::non_neg_integer()}].
 -type size_info() :: [{Name::atom(), Size::non_neg_integer()}].
 -type partition_info() :: [
-    {Partition::atom(), Partition::binary()} |
-    {DocCount::atom(), DocCount::non_neg_integer()} |
-    {DocDelCount::atom(), DocDelCount::non_neg_integer()} |
-    {Size::atom(), size_info()}
+    {partition, Partition::binary()} |
```

Comment:
db_name is missing

Reply:
we don't return the db name at this level. We do that at the fabric level.